js/src/components/paid-ads/ads-campaign.js

```diff
@@ -101,10 +108,16 @@ export default function AdsCampaign( {
 			<StepContentFooter>
 				<AppButton
 					isPrimary
-					disabled={ ! isValidForm }
+					disabled={
+						! isValidForm ||
+						( trackingContext === 'setup-ads' &&
```

Comment:
@kt-12 Why we need to check for `trackingContext` 🤔 ? Irrespective of the context, shouldn't the button be disabled if the form is not valid and billing has not been approved?

Reply:
This form is used in a different place too where billing is not involved. [ref](https://github.com/woocommerce/google-listings-and-ads/pull/2577#discussion_r1747864917)

Reply:
Since this is revisit this in 2535 with a holistic view of billing scenarios, I'll close this with the current condition.

Reply:
Sounds good @kt-12 .